src/framework/application.js

```diff
@@ -724,6 +757,30 @@ Object.assign(pc, function () {
             this._sceneRegistry.loadSceneSettings(url, callback);
         },
 
+        /**
+         * @callback pc.Application.loadSceneCallback
+         * @description Callback function used by {@link pc.Application#loadScene}.
+         * @param {String|Null} err The error message in the case where the loading or parsing fails.
+         * @param {pc.Entity} [entity] The loaded root entity if no errors were encountered.
+         */
+
+        /**
+         * @function
+         * @name pc.Application#loadScene
```

Comment:
I have a feeling pc.Application#loadScene isn't used anywhere and it may be best to keep it private. @daredevildave - can you state your preference?

Reply:
It is supposed to be used when using the engine programmatically. What other API is there when we need to load a scene and apply its settings? Chaining `loadSceneHierarchy` and `loadSceneSettings` will just unnecessarily waste load-time.

Reply:
If `loadSceneSettings` already does this, then the name is quite counter-intuitive and the behavior is not symmetric with `loadSceneHierarchy`.

Reply:
Well, in lieu of @daredevildave's response, I've made pc.Application#loadScene private for now so I can get this PR merged. 😄 We can always make public later.